Add eligibility section to application details #165294402

software-project opened the pull request:
https://www.pivotaltracker.com/story/show/165294402

software-project committed:
[#165294402] Add eligibility section to application details

software-project committed:
Merge branch 'master' into features/add-eligibility-section-to-appl-details-#165294402

software-project committed:
[#165294402] Add tests for eligibility section on paper application

akegan commented:
@software-project section should go below primary applicant. I'll move it for you tomorrow morning

It'd be great if we could put all 3 checkbox questions in the first column if that's possible!

akegan committed:
[#165294402] Move eligibility below primary applicant

akegan committed:
[#165294402] Show first 3 eligibility fields on the same side

Add ability to choose which fields go on which side of application view content card

akegan commented:
this PR is ready for review by someone other than me! (could be martin too but he's OOO)

software-project reviewed (commented):
Nice update for splitting. Looks good!
